client/components/vote-escrow/LockupsTable.tsx

```diff
@@ -0,0 +1,166 @@
+            reloadLockups();
+          },
+        },
+      ],
+    });
+  };
+
+  return (
+    <>
+      <table className="table table-compact w-full mb-4">
+        <thead>
+          <tr>
+            <th>OGV</th>
+            <th>Time remaining</th>
+            <th>Lockup ends</th>
+            <th>veOGV</th>
+          </tr>
+        </thead>
+        <tbody>
+          {lockups.length > 0 &&
+            lockups.map((lockup) => {
+              return (
+                <>
+                  <tr key={`${lockup.lockupId}`}>
+                    <td>
+                      <TokenAmount amount={lockup.amount} />
+                    </td>
+                    <td>
+                      <TimeToDate epoch={lockup.end} />
+                    </td>
+                    <td>{moment.unix(lockup.end).format("MMM D, YYYY")}</td>
+                    <td>
+                      <TokenAmount amount={lockup.points} />
+                    </td>
+                    <td>
+                      <Link
+                        className="btn rounded-full btn-sm btn-primary"
+                        href={`/stake/${lockup.lockupId}`}
+                      >
+                        Extend
+                      </Link>
+                    </td>
+                    <td>
+                      <Button
+                        white
+                        small
+                        disabled={blockTimestamp < lockup.end}
+                        onClick={() => handleUnlock(lockup.lockupId)}
+                      >
+                        Unstake
+                      </Button>
+                    </td>
+                    <td>
+                      <button
+                        className="flex-shrink-0 flex items-center justify-center w-4 p-0"
+                        onClick={() => {
+                          setModalLockup(lockup);
+                          setShowTxModal(true);
+                        }}
+                      >
+                        <EtherscanIcon />
+                      </button>
+                    </td>
+                  </tr>
+                </>
+              );
+            })}
+        </tbody>
+      </table>
+      <Modal show={showTxModal} handleClose={() => setShowTxModal(false)}>
+        <h3 className="mb-4 text-lg">Transaction history</h3>
+        <table className="table table-compact w-full">
+          <thead>
+            <th>Transaction time</th>
+            <th>Event</th>
+            <th>Transaction hash</th>
+          </thead>
+          <tbody>
+            {modalLockup.transactions.map((transaction) => {
+              return (
+                <tr key={transaction.hash}>
+                  <td>
+                    {moment(transaction.createdAt).format(
+                      "MMM D, YYYY, HH:mm:ss"
+                    )}
+                  </td>
+                  <td>{transaction.event}</td>
+                  <td>
+                    <Link
+                      href={
```

Comment:
nit: would probably be better for these to open in a new window

Reply:
> nit: would probably be better for these to open in a new window

Good catch, I have a prop for that ;)